src/plot_api/edit_types.js

```diff
@@ -0,0 +1,84 @@
+/**
+* Copyright 2012-2017, Plotly, Inc.
+* All rights reserved.
+*
+* This source code is licensed under the MIT license found in the
+* LICENSE file in the root directory of this source tree.
+*/
+
+'use strict';
+
+var isPlainObject = require('../lib/is_plain_object');
+
+module.exports = {
```

Comment:
Ah sure, I can do that. I was worried about circular deps if it lived in either `plot_api.js` or `plot_schema.js` but looks like it should work fine in `helpers.js`